### PR TITLE
Rename `--remote_workdir` and update CHANGELOG

### PR DESCRIPTION
**Pull request by mckern**
This PR changes `--remote_workdir` to `--remote-workdir`, updates the appropriate tests, and fixes a couple of typos. THEN it updates the CHANGELOG for a pending 0.11.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,21 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+### Added
+- A new Platform DSL method, `shell`, has been added. This allows a user to define a custom shell or path to a specific shell for a given platform.
+- A new Platform DSL method, `dist`, has been added. This allows a user to define a custom `%dist` parameter for a given platform's packaging format (if it's supported).
```
Comment (mwaggett) [inline CHANGELOG.md]: This was already released in 0.9.3.

**Comment by mckern** [comment]
@underscorgan I was going to do that after this changelog goes in, but I can append another commit that makes it official.

**Comment by underscorgan** [comment]
@mckern yeah, I think it would be better to do it all together.

**Comment by mckern** [comment]
@underscorgan updated

**Pull request merged**